yt_dlp/extractor/rtvslo.py

```diff
@@ -0,0 +1,150 @@
+from .common import InfoExtractor
+from ..utils import ExtractorError, traverse_obj, parse_duration, unified_timestamp
+
+
+class RTVSLOIE(InfoExtractor):
+    IE_NAME = 'rtvslo.si'
+    _VALID_URL = r'https?://(?:(?:365|4d)\.rtvslo.si/arhiv/[^/?#&;]+|(?:www\.)?rtvslo\.si/rtv365/arhiv)/(?P<id>(\d+))'
+    _API_BASE = 'https://api.rtvslo.si/ava/{}/{}?client_id=82013fb3a531d5414f478747c1aca622'
+    _GEO_COUNTRIES = ['SI']
+    _TESTS = [
+        {
+            'url': 'https://www.rtvslo.si/rtv365/arhiv/174842550?s=tv',
+            'info_dict': {
+                'id': '174842550',
+                'ext': 'mp4',
+                'release_timestamp': 1643140032,
+                'upload_date': '20220125',
+                'series': 'Dnevnik',
+                'thumbnail': 'https://img.rtvcdn.si/_up/ava/ava_misc/show_logos/92/dnevnik_3_wide2.jpg',
+                'description': 'md5:76a18692757aeb8f0f51221106277dd2',
+                'timestamp': 1643137046,
+                'title': 'Dnevnik',
+                'series_id': '92',
+                'release_date': '20220125',
+                'duration': 1789,
+            },
+        }, {
+            'url': 'https://365.rtvslo.si/arhiv/utrip/174843754',
+            'info_dict': {
+                'id': '174843754',
+                'ext': 'mp4',
+                'series_id': '94',
+                'release_date': '20220129',
+                'timestamp': 1643484455,
+                'title': 'Utrip',
+                'duration': 813,
+                'thumbnail': 'https://img.rtvcdn.si/_up/ava/ava_misc/show_logos/94/utrip_1_wide2.jpg',
+                'description': 'md5:77f2892630c7b17bb7a5bb84319020c9',
+                'release_timestamp': 1643485825,
+                'upload_date': '20220129',
+                'series': 'Utrip',
+            },
+        }, {
+            'url': 'https://365.rtvslo.si/arhiv/il-giornale-della-sera/174844609',
+            'info_dict': {
+                'id': '174844609',
+                'ext': 'mp3',
+                'series_id': '106615841',
+                'title': 'Il giornale della sera',
+                'duration': 1328,
+                'series': 'Il giornale della sera',
+                'timestamp': 1643743800,
+                'release_timestamp': 1643745424,
+                'thumbnail': 'https://img.rtvcdn.si/_up/ava/ava_misc/show_logos/il-giornale-della-sera_wide2.jpg',
+                'upload_date': '20220201',
+                'bitrate': 128000,
+                'release_date': '20220201',
+            },
+
+        }, {
+            'url': 'https://4d.rtvslo.si/arhiv/dnevnik/174842550',
+            'only_matching': True
+        }
+    ]
+
+    def _real_extract(self, url):
+        v_id = self._match_id(url)
+        meta = self._download_json(self._API_BASE.format('getRecordingDrm', v_id), v_id).get('response')
```

Comment:
Is this supposed to be fatal or not?
```suggestion
        meta = self._download_json(self._API_BASE.format('getRecordingDrm', v_id), v_id)['response']
```
OR

```suggestion
        meta = self._download_json(self._API_BASE.format('getRecordingDrm', v_id), v_id, fatal=False).get('response') or []
```